lms/services/jstor.py

```diff
@@ -174,6 +209,31 @@ def format_uri(template: str, *params: str):
     return template.format(*encoded)
 
 
+def _strip_html_tags(html: str) -> str:
+    """Extract plain text from a string which may contain HTML formatting tags."""
+    text = ""
+
+    # Extract text nodes using HTMLParser. We rely on it being tolerant of
+    # invalid markup.
+    #
+    # See https://bugs.python.org/issue31844 for abstract-method issue.
+    class Parser(HTMLParser):  # pylint:disable=abstract-method
+        def handle_data(self, data: str):
+            nonlocal text
+            text += data
+
+    parser = Parser()
+    parser.feed(html)
+    parser.close()
+
+    return text
+
+
+def _normalize_whitespace(val: str) -> str:
+    """Convert all whitespace to single spaces and remove leading/trailing spaces."""
+    return " ".join(val.split())
+
```

Comment:
The above two functions are fairly generic string-processing helpers. Where should they live if we wanted to reuse them in future? The lms app doesn't have a `util` package like h does.